### ISSUE TITLE
NumberFormatException on server start-up

### ISSUE THREAD

**Issue by SlimeDog** [SirBlobman/SirBlobmanAPI#14]
**Describe the bug**
Error on server start-up.

**To Reproduce**
- Install SirBlobmanCore 2.3.2-b108
- Start server
- Review log for error

**Expected behavior**
No errors.

**Screenshots**
```
[21:46:16] [Server thread/INFO]: [SirBlobman Core] Enabling SirBlobmanCore v2.3.2
[21:46:16] [Server thread/ERROR]: Error occurred while enabling SirBlobmanCore v2.3.2 (Is it up to date?)
java.lang.NumberFormatException: For input string: ""
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:678) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:786) ~[?:?]
        at com.github.sirblobman.api.utility.VersionUtility.getMinorVersion(VersionUtility.java:66) ~[?:?]
        at com.github.sirblobman.api.core.CorePlugin.registerListeners(CorePlugin.java:136) ~[?:?]
        at com.github.sirblobman.api.core.CorePlugin.onEnable(CorePlugin.java:49) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.18-pre5-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.18-pre5-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.18-pre5-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:525) ~[spigot-1.18-pre5-R0.1-SNAPSHOT.jar:3310-Spigot-c07753b-a58fde4]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:439) ~[spigot-1.18-pre5-R0.1-SNAPSHOT.jar:3310-Spigot-c07753b-a58fde4]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:232) ~[spigot-1.18-pre5-R0.1-SNAPSHOT.jar:3310-Spigot-c07753b-a58fde4]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:993) ~[spigot-1.18-pre5-R0.1-SNAPSHOT.jar:3310-Spigot-c07753b-a58fde4]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-pre5-R0.1-SNAPSHOT.jar:3310-Spigot-c07753b-a58fde4]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```

**Versions**
```
>version
[22:09:35] [Server thread/INFO]: This server is running CraftBukkit version 3310-Spigot-c07753b-a58fde4 (MC: 1.18 Pre-release 5) (Implementing API version 1.18-pre5-R0.1-SNAPSHOT)
[22:09:35] [Server thread/INFO]: You are running the latest version
>version SirBlobmanCore
[22:09:41] [Server thread/INFO]: SirBlobmanCore version 2.3.2
[22:09:41] [Server thread/INFO]: A core plugin with a lot of useful classes for other projects.
[22:09:41] [Server thread/INFO]: Author: SirBlobman
```

**config.yml**
```
# Set this to false to prevent the update checker from running.
# Default: true
update-checker: false

# Set this to true if you are having issues and wish to see more console output.
# Default: false
debug-mode: false

# Set this to true to enable the command logger.
# Default: false
command-logger: false

# Set this to true to cache the language for a player when they join.
# When this is false, the plugin will wait until the first time a translated message is sent to the player.
# Default: true
cache-language-on-join: true

# Set this to 'false' to keep the cached language for players that have already left the server.
# This may cause confusion for players that relog to reset their language.
cache-language-remove-on-quit: true

# Set this to 'false' to ignore client-side language changes from players.
# This may cause confusion for players that actually want a different language.
cache-language-update-on-change: true
```

**Comment by SlimeDog** [comment]
Fixed in SirBlobmanCore 2.3.2-b112

**Comment by SlimeDog** [comment]
Spigot 1.18-pre8
SirBlobmanCore 2.3.3-b112
```
[18:02:43] [Server thread/ERROR]: Error occurred while enabling SirBlobmanCore v2.3.3-Dev (Is it up to date?)
java.lang.NumberFormatException: For input string: ""
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:678) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:786) ~[?:?]
        at com.github.sirblobman.api.utility.VersionUtility.getMinorVersion(VersionUtility.java:66) ~[?:?]
        at com.github.sirblobman.api.core.CorePlugin.registerListeners(CorePlugin.java:136) ~[?:?]
        at com.github.sirblobman.api.core.CorePlugin.onEnable(CorePlugin.java:49) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.18-pre8-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.18-pre8-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.18-pre8-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:525) ~[spigot-1.18-pre8-R0.1-SNAPSHOT.jar:3312-Spigot-7840c2a-5a39a23]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:439) ~[spigot-1.18-pre8-R0.1-SNAPSHOT.jar:3312-Spigot-7840c2a-5a39a23]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:232) ~[spigot-1.18-pre8-R0.1-SNAPSHOT.jar:3312-Spigot-7840c2a-5a39a23]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:993) ~[spigot-1.18-pre8-R0.1-SNAPSHOT.jar:3312-Spigot-7840c2a-5a39a23]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-pre8-R0.1-SNAPSHOT.jar:3312-Spigot-7840c2a-5a39a23]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```

**Comment by SirBlobman** [comment]
Thanks for reporting this issue. It should be fixed in build 113 and higher.

**Comment by SlimeDog** [comment]
Confirmed fix with SBC 2.3.3-b113 on Spigot 1.18-pre8-3315. Thanks.
